contract/src/test/java/io/neow3j/contract/NefFileTest.java

```diff
@@ -19,6 +20,18 @@
     @Rule
     public ExpectedException expectedException = ExpectedException.none();
 
+    @Test
+    public void newNefFile() {
+        byte[] script = Numeric.hexStringToByteArray(
+                "5700020c0548656c6c6f0c05576f726c642150419bf667ce41e63f18841140");
+        NefFile nef = new NefFile("neon", new Version(3, 0, 0, 0), script);
```

Comment:
minor thing: the version is "fixed"? Like, are there just some possible values for it? If yes, create a constant class and put static values there.

Reply:
These are exactly the values needed to match the reference script. I usually don't put those in constants since they are specific to the test.